modules/ossm-configuring-jaeger-existing-v1x.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/ossm-custom-resources.adoc
+
+[id="ossm-configuring-jaeger-existing-v1x_{context}"]
+= Connecting to an existing Jaeger instance
+
+In order for the SMCP to connect to an existing Jaeger instance, the following must be true:
+
+* The Jaeger instance is deployed in the same namespace as the control plane, for example, into the `istio-system` namespace.
+
+* To enable secure communication between services, you should enable the oauth-proxy, which secures communication to your Jaeger instance, and make sure the secret is mounted into your Jaeger instance so Kiali can communicate with it.
+
+* To use a custom or already existing Jaeger instance, set `spec.istio.tracing.enabled` to "false" to disable the deployment of a Jaeger instance.
+
+* Supply the correct jaeger-collector endpoint to Mixer by setting `spec.istio.global.tracer.zipkin.address` to the hostname and port of your jaeger-collector service. The hostname of the service is usually `<jaeger-instance-name>-collector.<namespace>.svc.cluster.local`.
+
+* Supply the correct jaeger-query endpoint to Kiali for gathering traces by setting `spec.istio.kiali.jaegerInClusterURL` to the hostname of your jaeger-query service - the port is normally not required, as it uses 443 by default. The hostname of the service is usually  `<jaeger-instance-name>-query.<namespace>.svc`.
```

Comment:
@JStickler We have to maintain the same format across the doc. I see an `svc` entry here.

Reply:
Good catch!  Done.